target_chains/cosmwasm/tools/src/chains-manager/injective.ts

```diff
@@ -0,0 +1,231 @@
+import {
+  ChainGrpcAuthApi,
+  MsgExecuteContract,
+  MsgInstantiateContract,
+  MsgMigrateContract,
+  MsgStoreCode,
+  MsgUpdateAdmin,
+  Msgs,
+  PrivateKey,
+  TxGrpcClient,
+  TxResponse,
+  createTransactionFromMsg,
+} from "@injectivelabs/sdk-ts";
+import {
+  ChainExecutor,
+  ExecuteContractRequest,
+  ExecuteContractResponse,
+  InstantiateContractRequest,
+  InstantiateContractResponse,
+  MigrateContractRequest,
+  MigrateContractResponse,
+  StoreCodeRequest,
+  StoreCodeResponse,
+  UpdateContractAdminRequest,
+  UpdateContractAdminResponse,
+} from "./chain-executor";
+import { DEFAULT_GAS_PRICE } from "@injectivelabs/utils";
+import assert from "assert";
+
+export class InjectiveExecutor implements ChainExecutor {
+  private readonly wallet: PrivateKey;
+  private readonly chainId = "injective-888";
+  private readonly gasMultiplier = 1.3;
+  private readonly gasPrice = DEFAULT_GAS_PRICE;
+
+  constructor(
+    private readonly grpcEndpoint: string,
+    readonly mnemonic: string
+  ) {
+    this.wallet = PrivateKey.fromMnemonic(mnemonic);
+  }
+
+  private getAddress(): string {
+    return this.wallet.toBech32();
+  }
+
+  private async signAndBroadcastMsg(msg: Msgs): Promise<TxResponse> {
```

Comment:
this method pretty much exists in the price pusher already, right? is there a way that we can share code between the deployer here and the pusher? Seems like the generic cosmwasm capabilities will be useful in the pusher as well.

(again, feel free to leave a TODO)